sql/updates/world/world_2018_11_24_02.sql

```diff
@@ -0,0 +1,177 @@
+-- Patch 2.0
+UPDATE `areatrigger_teleport` SET `patch`=11 WHERE  `id`=4535 AND `patch`=0;
```

Comment:
I think it'll be troublesome to check the patch enum each time we want to modify DB like this, but I don't have a good and simple proposition to fix that.

Reply:
I get what you mean but after a while you get used to the numbers